clients/src/main/resources/common/message/ConsumerGroupHeartbeatRequest.json

```diff
@@ -35,6 +37,8 @@
       "about": "-1 if it didn't change since the last heartbeat; the maximum time in milliseconds that the coordinator will wait on the member to revoke its partitions otherwise." },
     { "name": "SubscribedTopicNames", "type": "[]string", "versions": "0+", "nullableVersions": "0+", "default": "null", "entityType": "topicName",
       "about": "null if it didn't change since the last heartbeat; the subscribed topic names otherwise." },
+    { "name": "SubscribedTopicRegex", "type": "string", "versions": "1+", "nullableVersions": "0+", "default": "null",
```

Comment:
Hum... I wonder if `nullableVersions` should be `1+` too. Does it cause an issue if we change it?

Reply:
makes sense and it didn't cause any issue with the tests so changed.